app/frontend/src/components/TabBar/TabBar.tsx

```diff
@@ -15,7 +18,8 @@ export default function TabBar<T extends Record<string, React.ReactNode>>({
     setValue(newValue);
   };
   return (
-    <Tabs
+    <TabList
```

Comment:
What is the difference?

Reply:
This injects/handles the right aria props for accessibility according to the docs - https://material-ui.com/components/tabs/#experimental-api - so I thought why not use it. Otherwise I think it's the same as I believe `TabList` uses `Tabs`